src/features/apiDelegation/offered/ReceiptPage/ReceiptPage.module.css

```diff
@@ -1,3 +1,3 @@
-.listContainer {
-  margin-bottom: 40px;
+.list {
+  margin-bottom: 40px !important;
```

Comment:
!important is the absolutely last thing we should use to achieve the css we want as this in future will make it really hard for us to see which css-files that sets the styling. Did you explore other options?

Reply:
> This was the best way I could think of and !important isn't strictly bad, it exists for a reason,

Using !important is considered bad practise and should be the last resort after trying everything else. I will try to solve it another way.

Reply:
You can just wrap the list in a div and add margin to the div or add margin-top to the elements below the list to achieve the same margin

Reply:
Personally, I think the use of important in this case is a more elegant and clean solution than adding extra divs whose only purpose is adding margins.
It means far more clutter and I cannot see how the important will harm anything in this particular case

Reply:
It has it's use case for example for Wordpress where it's really hard to style the web page in other ways, it just doesn't have a use case here where we _can_ solve it in other ways. And bad practise code isn't always removed from the programming languages because it's the way you use it, that defines if it's bad practise or not.

If you don't like adding extra divs you can just apply margin-top to the elements below the list to achieve the same result.

Reply:
Also, taking potential problems vs code elegancy I think creating solutions that doesn't create potential problems outweighs elegancy by far.

I worked in a code base where it was used way too much important and that made it really hard to style the web site like you wanted because css priority didn't mean a thing there and you would just need to guess where the css containing the importants would be

Reply:
Also when googling "css important" you will find almost everybody saying it should only be used as last resort. Even w3 has it stated in the article about the property 😅

<img width="1205" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/c2b4e85e-51b5-4395-989b-7671aecb8268">

Reply:
That being said, I can see that we disagree on this point, and this hill is not one I'm gonna die on.
So I made the change to a wrapping-div anyway, which will be up in a minute